ComfortZoneApp/app/src/main/java/com/example/comfortzone/InitialComfortActivity.java

```diff
@@ -110,11 +110,11 @@ public void done(ParseException e) {
                 } else {
 
                     if (level == 0) {
-                        addEntry(entryZero);
+                        tracker.addEntry(entryZero);
                     } else if (level == 5) {
-                        addEntry(entryFive);
+                        tracker.addEntry(entryFive);
                     } else if (level == 10) {
-                        addEntry(entryTen);
+                        tracker.addEntry(entryTen);
```

Comment:
ohh, nice catch, you already have the tracker, so we should have just add the entry there. However, 2 questions:
1. if you do this, don't you need to to call saveInBackground?
2. why couldn't you add entry to tracker before saveInBackground  one line 105 above?